starter-code/styles/modules.css

```diff
@@ -13,6 +13,9 @@
   /* TODO: Add styles for the icon-menu class (whether it be the
       hamburger font icon or an image of your choosing). It should be
       bigger, and off to the right. */
+  /*display: block;*/
```

Comment:
The TODO asked you to style your icon-menu class so that it is "off to the right." What attribute would you use to achieve this?